pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -173,7 +173,6 @@ static_resources:
                   timeout: 0.000s
           stat_prefix: "15011"
         name: envoy.http_connection_manager
-{{- if .ControlPlaneAuth }}
```

Comment:
I don't have all of the context here. Can you explain what's going on briefly?

Reply:
Port 15010 is 'insecure' GRPC, we want 15011 to be MTLS enabled at all times, independent of ControlPlaneAuth: otherwise it will be impossible to do gradual upgrade from insecure to secure. 

We had a similar problem with the 15003 http port in v1, which was also on/off based on controlplaneauth. Now we use 15005 as v1+MTLS, 15007 (and 8080) as v1+insecure.

Reply:
Ok thanks.  sgtm.